### DISCUSSION TITLE
Dependency track projects getting deactivated automatically

### DISCUSSION THREAD

**Discussion by ndbg2024** [DependencyTrack/dependency-track#4392] [Q&A]
I am seeing projects in dependency track getting deactivated automatically once they remain idle for a certain period of time (i.e. if they are not accessed for a certain amount of time (how much??? not sure about that as well). Have searched on internet and in the dependency track documentation as well but couldn't find anything on this issue. So, could someone provide some help or insights into this and also a fix if they have it. It would be really appreciated.

**Comment by nscuro** [comment]
Projects are not deactivated automatically.
